test/units/cli/test_galaxy.py

```diff
@@ -114,13 +114,13 @@ def tearDown(self):
 
     def test_init(self):
         galaxy_cli = GalaxyCLI(args=self.default_args)
-        self.assertTrue(isinstance(galaxy_cli, GalaxyCLI))
+        self.assertIsInstance(galaxy_cli, GalaxyCLI)
```

Comment:
Since you're updating the assertions, why not use `assert` instead? That will mean one less thing to change in the future when these tests are updated to not use `unittest.TestCase`. I'm not suggesting doing that conversion now, just using `assert` where the assertions are already being rewritten in this PR.